src/ResourceManager/LogicApp/Commands.LogicApp/Cmdlets/IntegrationAccount/UpdateAzureIntegrationAccountMapCommand.cs

```diff
@@ -109,30 +109,43 @@ public override void ExecuteCmdlet()
                 this.Name,
                 this.MapName);
 
+            var integrationAccountMapCopy = new IntegrationAccountMap(mapType: integrationAccountMap.MapType,
+                id: integrationAccountMap.Id,
+                name: integrationAccountMap.Name,
+                type: integrationAccountMap.Type,
+                location: integrationAccountMap.Location,
+                tags: integrationAccountMap.Tags,
+                parametersSchema: integrationAccountMap.ParametersSchema,
+                createdTime: integrationAccountMap.CreatedTime,
+                changedTime: integrationAccountMap.ChangedTime,
+                content: integrationAccountMap.Content,
+                contentLink: null,
+                metadata: integrationAccountMap.Metadata);
+
             if (!string.IsNullOrEmpty(this.MapFilePath))
             {
-                integrationAccountMap.Content = CmdletHelper.GetContentFromFile(this.TryResolvePath(this.MapFilePath));
+                integrationAccountMapCopy.Content = CmdletHelper.GetContentFromFile(this.TryResolvePath(this.MapFilePath));
             }
 
             if (!string.IsNullOrEmpty(this.MapDefinition))
             {
-                integrationAccountMap.Content = this.MapDefinition;
+                integrationAccountMapCopy.Content = this.MapDefinition;
                 CmdletHelper.GetContentFromFile(this.TryResolvePath(this.MapFilePath));
             }
 
             if (!string.IsNullOrEmpty(this.ContentType))
             {
-                integrationAccountMap.ContentType = this.contentType;
+                integrationAccountMapCopy.ContentType = this.contentType;
             }
 
             if (!string.IsNullOrEmpty(this.MapType))
             {
-                integrationAccountMap.MapType = (MapType) Enum.Parse(typeof(MapType), this.MapType);
+                integrationAccountMapCopy.MapType = (MapType) Enum.Parse(typeof(MapType), this.MapType);
```

Comment:
Fix spacing in this file as you fix the bug. There should be no space character after closing parenthesis when type-casting. (Other lines:) There should be no space after the typeof keyword and before the opening parenthesis.